src/AzureSqlActionHelper.ts

```diff
@@ -107,6 +89,32 @@ export default class AzureSqlActionHelper {
         return filePath;
     }
 
+    public static getSqlpackageActionTypeFromString(action: string): SqlPackageAction {
+        // Default to Publish if not specified
+        if (!action) {
+            return SqlPackageAction.Publish;
+        }
+
+        switch (action.trim().toLowerCase()) {
+            case 'publish':
+                return SqlPackageAction.Publish;
+            // case 'extract':
```

Comment:
in this intentional? why the lines are commented?

Reply:
Yes only the Publish-like actions are supported for now